### ISSUE TITLE
What is the first thing to do?

### ISSUE THREAD

**Issue by fartzy** [grizzlypeaksoftware/flask-auth-service#1]
Hi in the route "/client"  for a post method,  the first thing the code  checks the database against the token given in the authorization header.        
        
        # verify the token 
        authorizationHeader = request.headers.get('authorization')
        token = authorizationHeader.replace("Bearer ", "")
        verification = authModel.verify(token)

But then the rest of the code goes on to insert a new record after hashing the authorization token.  So an initial insertion can never happen because the verification will fail.  Am I missing something?

**Comment by grizzlypeaksoftware** [comment]
Agreed!  The code here does not allow for the ability to create a user on the first insertion. 

In the article that goes along with this code, we first create the client route before setting up authorization, and then I added in the authorization later.

Check out the article.
https://www.grizzlypeaksoftware.com/articles?id=5SCpQMgookgKNtupzNHg9K

I will think about setting up a way to do a first-time insertion of an admin client and will put in an update.  

In the meantime, you can simply comment out the bit of code that validates the token for the first time commit in order to create your first admin client.

Thank you for using my code!

Shane

**Comment by fartzy** [comment]
Ah I see I got confused when I looked at the code and didn't comment out the verification.  Thanks for uploading it man, good work!
